BLSQ/iaso
# IA-3481: Reassign Forms to Projects (simplified)

Explain what problem this PR is resolving

Related JIRA tickets : [IA-3481](https://bluesquare.atlassian.net/browse/IA-3481)

## Self proofreading checklist



## Changes

Reassigning forms to their proper projects

## How to test

From setuper folder, launch setuper via `python3 setuper.py --additional_projects` to generate test data. Then login with the created account and open [Forms page](http://localhost:8081/dashboard/forms/list/).

You should see projects:
 - Main Project: with 1 form
 - Georegistry/Géoregistre: 1 form
 - Planning: 1 form
 - Vaccination: 2 forms.

## Print screen / video

![Iaso-Forms](https://github.com/user-attachments/assets/be719545-5a40-4237-bf0e-d587550bb2a4)




[IA-3481]: https://bluesquare.atlassian.net/browse/IA-3481?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
